ImisRestApi/Data/ImisBasePayment.cs

```diff
@@ -802,5 +802,36 @@ public List<ReconciliationItem> ProvideReconciliationData(ReconciliationRequest
 
             return result;
         }
+
+        public List<String> GetProductsSPCode()
+        {
+
+            var getProductsSPCodes = @"SELECT AccCodePremiums FROM tblProduct WHERE AccCodePremiums LIKE '%SP[0-9{3}]%' AND ValidityTo is NULL";
```

Comment:
I think it should be like that "SP[0-9]{3}" because % will look for data before and after which is not required